### DISCUSSION TITLE
Pending student pack application

### DISCUSSION THREAD

**Discussion by Huiting-coding** [community/community#73023] [GitHub Education]
### Hi everyone,

hello!
The bootcamp school (le wagon) i'm in is a Github campus partner. 
The school acutally provide a link to fast-verify the student pack, however, without noticing this, i applied for the student pack through the  education.github.com myself on Spetember 8th, but the application is still pending. Right now i find myself in a dead loop:(
I tried to click the link provide by my school:
![image](https://github.com/community/community/assets/88573186/c9b971d5-c25d-468b-b0db-63bb4ce74c5c)
it shows succesful.
but if I went on signing in, i will be lead to this page:
<img width="1231" alt="70ad0727144bb7da5c00018edcc84a6" src="https://github.com/community/community/assets/88573186/6bfacce9-e02f-4289-a26b-39cb9efd81b5">
after i filling the information,i can not continue, but my email address should be fine.
<img width="966" alt="1aaab33e428ea786e3ecd7aae3d964b" src="https://github.com/community/community/assets/88573186/7510876a-7307-48e9-b581-99d440fc7b44">

also, if I return to the homepage for Education, it says they are still processing my application.
<img width="1258" alt="af5f689b948f9407d16f6453014a7e8" src="https://github.com/community/community/assets/88573186/29fe4fd1-4e6b-460c-8e91-f6eb5bd1be12">

Also, I tried to check the application status, but i'm then stuck in step 2 again...
Is it possible that github support can just cancel my application on September,  and i can directly redeem the pack by the link privided by Le wagon? or it's another problem?

Thanks in advance!!

**Comment by SayedMohsen635** [comment]
me too and I need a solution
